op-node/rollup/derive/progress.go

```diff
@@ -23,7 +23,7 @@ func (pr *Progress) Update(outer Progress) (changed bool, err error) {
 	}
 	if pr.Closed {
 		if outer.Closed {
-			if pr.Origin != outer.Origin {
+			if pr.Origin.ID() != outer.Origin.ID() {
```

Comment:
Fine to me to do the comparison like this, but is there even a functional difference, other than a possible (not sure) performance gain?

Reply:
I think I was running into issues with the tests by not having a full eth.L1BlockRef instead of the IDs

Reply:
TBH all you need is the hash for equality checks